providers/cncf/kubernetes/docs/changelog.rst

```diff
@@ -28,6 +28,34 @@ Changelog
 ---------
 
 
+10.11.0
+.......
+
+.. note::
+    This release of provider is only available for Airflow 2.11+ as explained in the
+    Apache Airflow providers support policy <https://github.com/apache/airflow/blob/main/PROVIDERS.rst#minimum-supported-version-of-airflow-for-community-managed-providers>_.
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Fix Account for job- prefix when truncating job names (#58391)``
+* ``KubernetesPodOperator: Rework of Kubernetes API retry behavior (#58397)``
+* ``Fix KuberetesPodTriggerer use correct parameter name to read pod logs. (#58489)``
+* ``KubernetesPodOperator pushes XCom on successful execution. (#58488)``
+* ``KubernetesPodOperator PodManager retries during create pod on too many requests error (#58033)``
+* ``Add JSON serialization for kubeconfig in AsyncKubernetesHook (#57169)``
```

Comment:
Wasn't sure about this one - looked like a feature - but its' a bugfix indeed.

Reply:
Yes, is a bug fix